driver/src/main/java/org/kaazing/k3po/driver/internal/netty/bootstrap/file/FileChannel.java

```diff
@@ -138,6 +138,9 @@ private static MappedByteBuffer mapFile(URI fileAddress, String mode, long size)
                 break;
             case "rw":
                 mapMode = READ_WRITE;
+                // parent directory must exist to create file
+                File parentDir = location.getParentFile();
+                parentDir.mkdirs();
```

Comment:
Throwing the exception doesn't get us a solution unless theres another way to create the parent directory, which would imply adding that capability to the K3PO scripting language (ugh!) or requiring the specification tests to have more than just `k3po.finish()` (double ugh!). :smile:

I think it would be good for you to add a different test case to verify the permissions behavior with a negative test.